src/components/Card.jsx

```diff
@@ -0,0 +1,136 @@
+import Arrow from "../assets/icon-arrow.svg";
+import { useEffect, useState } from "react";
+
+import FormInput from "./FormInput";
+import AgeResults from "./AgeResults";
+const monthsUtil = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+const Card = () => {
+  const [day, setDay] = useState("");
```

Comment:
It would be ideal for initializing the dates with a number, to hint to TS that they must be numbers. But I also agree that the fields should appear empty. Maybe set them to zero and make the fields handle that particular case and not display anything?